package.json

```diff
@@ -8,6 +8,8 @@
     "@types/meteor": "1.4.76",
     "@types/node": "16.4.10",
     "@types/react": "17.0.15",
-    "eslint-config-vazco": "6.2.0"
+    "eslint-config-vazco": "6.2.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "prettier": "2.5.1"
```

Comment:
These two are implied by `eslint-config-vazco` if you'll use the new `package-lock.json` format. I'd do that in this PR.